ui/pages/mobile-sync/mobile-sync.component.js

```diff
@@ -408,7 +408,10 @@ export default class MobileSyncPage extends Component {
     const { password } = this.state;
 
     return (
-      <div className="new-account-import-form__buttons" style={{ padding: 30 }}>
+      <div
+        className="new-account-import-form__buttons"
```

Comment:
Ideally the mobile sync component would have its own set of css rules, but they are borrowed elswhere, hence the pre-existing `padding` overridee